index.html

```diff
@@ -3955,6 +4040,7 @@ <h2>
         <pre class="idl">
           [Constructor(DOMString type, optional PaymentRequestUpdateEventInit eventInitDict), SecureContext, Exposed=Window]
           interface PaymentRequestUpdateEvent : Event {
+            Promise&lt;PaymentAddress&gt; requestShippingAddress(AddressParts addressParts);
```

Comment:
```suggestion
            Promise&lt;PaymentAddress&gt; requestShippingAddress(FrozenArray<AddressParts> addressParts);
```

The description on line 4083 talks about a list of AddressParts.

Reply:
Note that here it should take a sequence... FrozenArrays are returned by methods, but generally not passed into a method call.  Will add a suggestion.